client/src/composables/__mocks__/config.ts

```diff
@@ -0,0 +1,44 @@
+import { vi } from "vitest";
+import { ref } from "vue";
+
+// Default mock config values that tests can override
+const mockConfig = ref({
+    allow_local_account_creation: true,
+    enable_oidc: false,
+    mailing_join_addr: "",
+    prefer_custos_login: false,
+    registration_warning_message: "",
+    server_mail_configured: false,
+    show_welcome_with_login: false,
+    terms_url: "",
+    welcome_url: "",
+    citation_bibtex: "",
+    toolbox_auto_sort: false,
```

Comment:
Maybe move this object to a const and then reuse it here and in resetMockConfig.

Reply:
Good call, will refactor